src/ui/public/notify/partials/toaster.html

```diff
@@ -1,6 +1,6 @@
 <div class="toaster-container">
   <ul class="toaster">
-    <li ng-repeat="notif in list" kbn-toast notif="notif">
+    <li ng-repeat="notif in list" kbn-toast notif="notif" data-test-subj="bootstrapToast">
```

Comment:
The old Bootstrap toasts are only used for errors now. This is a way to check for an error outcome.